package.json

```diff
@@ -2,25 +2,64 @@
   "name": "ember-router-scroll",
   "version": "0.5.0",
   "description": "Scroll to top with preserved browser history scroll position",
+  "keywords": [
+    "ember-addon",
+    "ember-router-scroll",
+    "popstateEvent",
+    "browser scroll"
+  ],
+  "license": "MIT",
+  "author": "Jason Farmer <jacefarm@gmail.com> and Benny C. Wong <bwong337@gmail.com>",
+  "contributors": [
+    {
+      "name": "Benny Wong",
+      "email": "bwong337@gmail.com"
+    },
+    {
+      "name": "Jason Farmer",
+      "email": "jacefarm@gmail.com"
+    },
+    {
+      "name": "Brian Cardarella",
+      "url": "https://github.com/bcardarella"
+    },
+    {
+      "name": "Arjan Singh",
+      "url": "https://github.com/arjansingh"
+    },
+    {
+      "name": "Brian Gonzalez",
+      "url": "https://github.com/briangonzalez"
+    },
+    {
+      "name": "Robert Wagner",
+      "url": "https://github.com/rwwagner90"
+    },
+    {
+      "name": "Scott Newcomer",
+      "url": "https://github.com/snewcomer"
+    }
+  ],
   "directories": {
     "doc": "doc",
     "test": "tests"
   },
-  "scripts": {
-    "build": "ember build",
-    "lint:js": "eslint ./*.js addon addon-test-support app config lib server test-support tests",
-    "start": "ember serve",
-    "test": "ember try:each",
-    "changelog": "lerna-changelog",
-    "deploy": "ember github-pages:commit --message \"Deploy gh-pages from commit $(git rev-parse HEAD)\"; git push; git checkout -"
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/dollarshaveclub/ember-router-scroll.git"
   },
   "bugs": {
     "url": "https://github.com/dollarshaveclub/ember-router-scroll/issues"
   },
   "homepage": "https://dollarshaveclub.github.io/ember-router-scroll/",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/dollarshaveclub/ember-router-scroll.git"
+  "scripts": {
+    "build": "ember build",
+    "lint:js": "eslint ./*.js addon addon-test-support app config lib server test-support tests",
+    "start": "ember serve",
+    "test": "ember test",
+    "changelog": "lerna-changelog",
```

Comment:
This is a bug. Please remove.

Reply:
This is not a bug, it's a changelog script, which we can easily still use by installing lerna-changelog.

Reply:
Could you run that script and paste the output? ~Thanks

Reply:
The script doesn't get run. lerna-changelog is a perfectly valid thing to use, if we decide to use it. It's 100% unrelated to this PR.